.gitmodules

```diff
@@ -67,3 +67,6 @@
 [submodule "shim"]
 	path = shim
 	url = https://github.com/facebook/buck2-shims-meta
+[submodule "backends/apple/coreml/third-party/json"]
+	path = backends/apple/coreml/third-party/json
+	url = https://github.com/nlohmann/json.git
```

Comment:
don't make this change in this PR

it should be a separate PR

nlohmann/json is not used in AOT, but for, runtime only

so technically we shouldn't be touching this at all for pip wheel for coreml.